### ISSUE TITLE
picons don't show up

### ISSUE THREAD

**Issue by AttilaSVK** [knackebrot/jellyfin-plugin-enigma2#23]
Set top box: Vu+ Zero 4K running OpenATV 7.0.20220902 
Enigma2 plugin version: 5.0.0.0
Jellyfin version: 10.8.13 running on Windows Server 2022 in ESXi 8.0 Update 2

Jellyfin clients tested: web client, Android TV client

The picons don't get pulled from the STB despite they are visible in OpenWebif. The picon resolution and format is 400x240px PNG with transparent backgrounds. The "Fetch picons from webinterface" option is checked, the picon path is not defined in the plugin settings.

Other apps (dream player TV for AndroidTV) can download picons from the box.

**Comment by knackebrot** [comment]
I probably use the same picons as you do, and it worked for me on both OpenATV and OpenPLI. Does your OpenWebIf require login?

**Comment by AttilaSVK** [comment]
Yes, it requires a login, because my box is on a different subnet. (placing it on the same subnet is not an option, because I can't install a satellite dish at my flat, so I access my STB over VPN)

**Comment by IamMattM** [comment]
@AttilaSVK 
Have you found a solution to the picon "pull-from-openwebif" issue ?

I think that it will be the same issue as for the Vuplus plugin used in Emby: missing SRP names png with 1_0_19 (for HD channels)
https://emby.media/community/index.php?/topic/18999-devvu-enigma2-live-tv-plugin/page/8/

**Comment by IamMattM** [comment]
I have just tried the latest jellyfin and plugin and it is showing the same issue whereby only the SD TV channels (1_0_1) SRP pngs are retieved. In my case only one channel is still in SD format while the others are all in HD (1_0_19) format but because I can't figure out how to acquire all those SRP named HD channel pngs, there are missing and therefore the plugin fails to load the HD channel picons.

![image](https://github.com/user-attachments/assets/c048dc37-b00c-420b-9e92-2bca1bbcc589)

**Comment by IamMattM** [comment]
My openwebif shows all of the picons for those HD channels above so it is as if the "retrive picons from openwebif" is not working.

**Comment by AttilaSVK** [comment]
@IamMattM in fact, I do use Chocholousek's picon plugin, so that might be the cause of the problem. On the other hand, I moved since Feburary, and I have my STB connected locally to the TV, no need for Jellyfin :)

**Comment by IamMattM** [comment]
I did try the Kodi add-on (https://github.com/kodi-pvr/pvr.vuplus) for enigma2 mentionned on the emby link above, and that add-on is able to extract the openwebif picons successfully from the same vu+ duo 4K, so whatever/however it is doing it it is able to retrieve the picons correctly...compared to the jellyfin add-on that seems to be looking for the SRP (service reference) 
names picon rather than the service name SNP picon.

 
![image](https://github.com/user-attachments/assets/0bf1f0fe-a875-4255-9559-2980870457d8)


    https://github.com/kodi-pvr/pvr.vuplus
